src/Leaves/Table.php

```diff
@@ -244,10 +245,12 @@ private function getSchemaColumns()
         // collection query before we run the filered one, causing the very slow down we were trying
         // to avoid.
 
-        if (!$this->model->unsearchedHtml) {
-            $this->model->collection->count();
+        $respository = $this->model->collection->getRepository(); 
+        if ($respository instanceof MySql) {
+            $namedParams = [];
```

Comment:
Any need for this explicit variable? Not seeing it used apart from the line below

Reply:
yeah - it's passed by reference so you need a variable. We don't plan to execute the query so we don't actually need the variable except that we must due to the pass by reference requirement. It's actually a symptom of the overall structure of the code not being completely correct I think but don't want to go too far with the restructuring.

Reply:
I'm confused what this whole bit is for? We don't appear to be using the sql statement result from the following line?